File: .idea/wxpusher.py
import requests
import os

def send_wxpusher_message(content, uid):
    app_token = os.getenv("WXPUSHER_APP_TOKEN")
    url = "https://wxpusher.zjiecode.com/api/send/message"
    data = {
        "appToken": app_token,
        "content": content,
        "uids": [uid],
        "contentType": 1
    }
    response = requests.post(url, json=data)
    return response.json()

File: backend/src/util/wxpusher.js
const axios = require('axios');

async function sendWxpusherMessage(content, uid) {
    const appToken = process.env.WXPUSHER_APP_TOKEN; // 从环境变量获取 App Token
    const url = "https://wxpusher.zjiecode.com/api/send/message";

    const data = {
        appToken,
        content,
        uids: [uid],
        contentType: 1
    };

    try {
        const response = await axios.post(url, data);
        return response.data;
    } catch (error) {
        console.error('WxPusher 发送失败:', error);
    }
}

module.exports = { sendWxpusherMessage };
